### DISCUSSION TITLE
Vents rotation issue (axis aligned restriction)

### DISCUSSION THREAD

**Discussion by AlanZandie** [firemodels/fds#12135] [General]
Dear all,

Good day, 

this is Dr. Zandie from the University of Birmingham, I have been trying to define an induction fan using a duct system that has got a deviation (angle) from the axes. This means it is not aligned with the cartesian axis and it makes a 45 degree with X and Y axes. So, basically, I first defined the fan system with nodes, outlets and the duct and tried to rotate it 45 degrees, but apparently, vents are allowed to rate only with multiple of 90 degrees.  For your convenience, I have attached a screenshot of the layout, where you can see two of the fans are inclined with angles with regard to the X and Y axes. I could not find a way around it, could you kindly help me come up with an idea to resolve this issue?
Best,

![Screenshot (2326)](https://github.com/firemodels/fds/assets/93418696/0945e7b3-af81-4207-a158-f619183220e5)

**Comment by drjfloyd** [comment]
Moved to discussion as this is not a bug report.

Do the fans have their supply outside the compartment?  If so you could use GEOM and just specify a volume flow where the exit is.  Another option is to align as best you can with the grid (set the orientation to the highest component of the flow) and use VEL_T (this would require more than one cell over the fan).

If the fans take supply inside the compartment, then I'd suggest align as best you can with the grid and use the tangential feature for HVAC. Like VEL_T this would require more than one cell over the fan.

**Comment by AlanZandie** [comment]
Hello,
The supply is attached to the induction fan (IF) and I have incorporated three of them as depicted in the figure above. I have attached the FDS code, and kindly note I want to redirect the flow of IF of 1 and 2 to be nearly 45 degrees long with the x and y axes. I also can use the Louvre option on the IF's outlets on the HVAC but the velocity kinda has a disturbed shape, and I dont think it is the best idea to do that. 
Could you please have a look,
thank you

[Project.fds.txt](https://github.com/firemodels/fds/files/12702075/Project.fds.txt)

**Comment by mcgratta** [comment]
I suggest you set up a very simple case to test your proposed strategy. The case you have posted is far too complicated to work with.

**Comment by drjfloyd** [comment]
You have 1 x 4 grid cells across the exit.  There is a limt to how well you can expect to resolve the flow angle with a small number of cells like that.  A 2x2 might yield better results but you should still not expect perfection.

**Comment by AlanZandie** [comment]
Dear all,
apologies for my late reply, I believe given that I'm using U V W velocities for my duct system, I can use the Louvre UVW option to specify that flow direction. Also, thank you for your suggestions, though I kept the meshes very course on some sites which weren't of interest. I only have one concern regarding the induction fan itself, although now we can define the flow direction at the outlet, yet again the geometry of the fan particularly its inlet and outlet must be aligned with the axes. I'm just a bit afraid this might impose erroneous outcomes in terms of how the flow would actually exert such a flow pattern. Anyhow, again, it's just a personal viewpoint and I'm not sure if I'm correct in thinking that way. 
thank you for your help
stay good

**Reply by drjfloyd** [reply]
Inlet orientation isn't like oulet.  The flow field for a jet fan is something like the image below. The outlet side is strongly directional; however, at the inlet the fan creates a pressure drop that pulls in air from all directions around the outlet. 

I suggest you make a simple model as Kevin suggested.  With that you can model just a fan in the open, have the flow be either grid aligned leaving the fan or using UVW, and investigate what the flow field looks like and how grid resolution impacts that. 


![image](https://github.com/firemodels/fds/assets/12799217/36fb61b4-56e2-4f04-ba36-31d270d0dd51)

**Comment by AlanZandie** [comment]
Thank you all for your help and clarification on this matter, I have created my model and it's working all right.
